### DISCUSSION TITLE
Bagaimana cara kirim pesan ke banyak nomer sekaligus dg sekali Request?

### DISCUSSION THREAD

**Discussion by ulilamri419** [kirimwa-id/forum#10] [Saran dan Ide]
Salam Mas Rio,

Dilihat dari dokumentasi developer kirimwa.id disitu belum menjelaskan bagaimana cara kirim ke banyak nomer seperti email to nya bisa banyak.
Apakah fitur ini belum tersedia? jika iya, mungkin akan sangat membantu sekali, jika ingin membuat semacam broadcast (fitur di WA) penggunaan dari fitur ini cocok sekali, jika mau launching product baru ke byk user tanpa perlu save nomer.

Terima kasih sudah menampung ide ini.
Salam
Amri

**Comment by rioastamal** [comment]
Halo mas @ulilamri419,

Terima kasih sarannya. Saya sedang mempertimbangkan bagaimana implementasinya dengan tetap memperhitungkan message quota dan response dari tiap-tiap pesan yang dikirim. Saya sudah memasukkannya ke dalam roadmap #11.
